feat, docs: enable custom markdown rendering to allow metadata (#568)
widgetti/solara: solara/website/pages/documentation/getting_started/content/02-installing.md

---

> title: Installing Solara
> description: Installation should be as easy as running pip install solara. More options are available.

Reviewer: ```suggestion
title: Installing Solara
description: Installation should be as easy as running pip install Solara. Read on for advanced setups.
```
